test/test-pkg-config.rb

```diff
@@ -233,9 +233,29 @@ def parse_requires(requires)
       @cairo.__send__(:parse_requires, requires)
     end
 
-    def test_broken_version
```

Comment:
Please don't remove this case. The `fribidi_required_dep` part (it's not a valid version number) is important.